main.go

```diff
@@ -114,6 +114,8 @@ func main() {
 	resolvers.Register("comment.create", collector.ConfiguredVars(config, collector.NewCommentResolver(authClient, witClient)), nil)
 	resolvers.Register("comment.update", collector.ConfiguredVars(config, collector.NewCommentResolver(authClient, witClient)), nil)
 	resolvers.Register("user.email.update", collector.ConfiguredVars(config, collector.NewUserResolver(authClient)), validator.ValidateUser)
+	resolvers.Register("invitation.team.noorg", collector.ConfiguredVars(config, nil), nil)
+	resolvers.Register("invitation.space.noorg", collector.ConfiguredVars(config, nil), nil)
```

Comment:
Passing a nil here `collector.ConfiguredVars(config,nil)`  causes the code to stumble upon https://github.com/fabric8-services/fabric8-notification/blob/master/collector/workitem.go#L40

which ends up causing the following error at runtime


```
INFO[2018-07-02 19:43:05] payload                                       identity_id=837f2447-2e42-4db9-9f32-817d4866178a pkg=github.com/fabric8-services/fabric8-notification/vendor/log.LogRequest.func1 raw="{\"data\":{\"attributes\":{\"custom\":{\"acceptURL\":\"localhost/accept\",\"inviter\":\"shane\",\"spaceName\":\"starwars\",\"teamName\":\"teamName\"},\"id\":\"8bccc228-bba7-43ad-b077-15fbb9148f7f\",\"type\":\"invitation.team.noorg\"},\"id\":\"2a35344c-43de-4767-bf3b-b951d22bf7df\",\"type\":\"notifications\"}}" req_id=K8Uq7zjyYF-1
INFO[2018-07-02 19:43:05] completed                                     action=send bytes=0 ctrl=NotifyController duration=0.24070999999999998 duration_unit=ms identity_id=837f2447-2e42-4db9-9f32-817d4866178a pkg=github.com/fabric8-services/fabric8-notification/vendor/log.LogRequest.func1 req_id=K8Uq7zjyYF-1 status=202
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x8bc382]

goroutine 36 [running]:
github.com/fabric8-services/fabric8-notification/collector.ConfiguredVars.func1(0xd5ecc0, 0xc420124e70, 0xc4204c1fb0, 0x24, 0x110, 0xa16440, 0x40fd01, 0x17f841f415310, 0x110, 0x110)
	/home/shbose/fabric8/notification/src/github.com/fabric8-services/fabric8-notification/collector/workitem.go:40 +0x72
github.com/fabric8-services/fabric8-notification/email.(*AsyncWorkerNotifier).do(0xc4204a2280, 0xd5ecc0, 0xc420124e70, 0xc4204a3f80, 0x15, 0xc4204c1fb0, 0x24, 0xc4204c1f80, 0xc4204a23c0, 0xc4204a3f80, ...)
	/home/shbose/fabric8/notification/src/github.com/fabric8-services/fabric8-notification/email/notifier.go:74 +0xa5
github.com/fabric8-services/fabric8-notification/email.(*AsyncWorkerNotifier).work(0xc4204a2280)
	/home/shbose/fabric8/notification/src/github.com/fabric8-services/fabric8-notification/email/notifier.go:66 +0x31a
created by github.com/fabric8-services/fabric8-notification/email.(*AsyncWorkerNotifier).start
	/home/shbose/fabric8/notification/src/github.com/fabric8-services/fabric8-notification/email/notifier.go:55 +0x4f
```

I used this payload to test

```
curl 'http://localhost:8081/api/notify' -X POST -H "Authorization: Bearer $token" -H 'Content-Type: application/json' -H 'Accept: application/json, text/plain, */*' --data-binary  '{"data":{"attributes":{"custom":{"teamName":"teamName","inviter":"shane","spaceName":"starwars","acceptURL":"localhost/accept"},"id":"8bccc228-bba7-43ad-b077-15fbb9148f7f","type":"invitation.team.noorg"},"ID":"2a35344c-43de-4767-bf3b-b951d22bf7df","type":"notifications"}}'
```